pkg/client/src/app/queries/volumes.ts

```diff
@@ -59,3 +40,54 @@ export const useFetchVolumes = (): IVolumeFetchState => {
     refetch,
   };
 };
+
+export const useCleanRepositoryMutation = ({ onSuccess, onError }) => {
+  const [processId, setProcessId] = useState(null);
+  const [stop, setStop] = useState(false);
```

Comment:
~I think we could use a more descriptive name than `stop` here like `isPollingDisabled` maybe? Also it seems like it should be defaulting to true?~

~Or maybe it would be more readable if we inverted it and called it something like `isPollingTaskStatus`, started it as false and set it true when the task starts and false when it's done.~

Actually, I think maybe you don't need this `stop` state at all. See below.